NameList/ContentView.swift

```diff
@@ -13,11 +13,18 @@ struct ContentView: View {
   let names: [NameType] = {
     var names: [NameType]!
     _ = NameImporter().importFrom(file: .test)
-      .collect()
+      .scan([NameType](), { (namesSoFar, currentName) -> [NameType] in
```

Comment:
So `collect()` waits until the source Publisher completes, which could be undesirable in the case of something that is slow or long-running.

`scan()`, by comparison, will simply amass all available items into an array, appending a new element onto the array when it arrives.